src/main.ml

```diff
@@ -114,7 +114,7 @@ let handle_connection fd =
       >>= fun sr_uuid ->
       with_attached_vdi sr_uuid vdi_rec.API.vDI_location (not vdi_rec.API.vDI_read_only)
         (fun filename -> 
-          with_block filename (Nbd_lwt_server.serve t (module Block))
+          with_block filename (Nbd_lwt_unix.Server.serve t (module Block))
         )
     finally
```

Comment:
there is a `try+lwt...finally`. Should we get rid of `lwt.syntax` from this module as well?